### DISCUSSION TITLE
"HTTP 419 Page Expired" during logout

### DISCUSSION THREAD

**Discussion by deviationist** [livewire/livewire#7725] [1. Help]
Livewire v3.3.5
Laravel v10.39.0
PHP 8.3.1

So I've recently built an app using Laravel Breeze that uses Livewire with class-based Volt-components. I'm using polling to keep my data up to date. But - during logout I sometimes get a "419 Page Expired"-error. My polling is set with 2 second intervals, and if the polling triggers just right after I click the logout button then I get an error. Ideally I'd like to stop Livewire from polling just when I click the logout button. This would prevent the polling-request from running and thus preventing the 419 error.

I've tested two ways to log out to see if that worked better:
1. Using the Laravel Breeze-way
Using Volt-components with a logout button that calls App\Livewire\Actions\Logout::class, then triggers a redirect to the login-page.

2. Logout using a form (the good old way)
I swapped the Volt-component with a form containing a submit-button which posts to a Logout-route, that invokes again App\Livewire\Actions\Logout::class, then redirects to the login-page.

With both of these alternatives I still sporadically the "419 Page Expired"-error.

So - how can I log out without the risk of this error occurring? I've yet to find how to stop the polling which would the preferred solution.

Kind regards
Robert S.

**Comment by joshhanley** [comment]
This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147
